### DISCUSSION TITLE
Paid help.

### DISCUSSION THREAD

**Discussion by moazam1** [postalserver/postal#1887] [Help with using Postal]
I am struggling with postal help so I was wondering if postal expert can help me with server configuration. I am struggling with IP pool. Our servers are behind NAT and postal is not picking up IP from pool. I must be missing something. Happy to pay hourly/fixed price. Please reach out if you can help. Thanks

**Comment by Ramon-LibertyPumps** [comment]
Did you ever get any guidance on this? I'm also having an issue with postal not using the address from the IP Pool.

**Comment by moazam1** [comment]
@Ramon-LibertyPumps nothing so far. Posted job on upwork but no luck.

**Reply by mitmelon** [reply]
Have you got this issue resolved... I can help

**Reply by moazam1** [reply]
@mitmelon not yet. I would love to get help.
